adding episode 24 (#16)
USRSE/rse-stories: _posts/2020-07-09-stephan-druskat.md

---
layout: post
title: "Language shapes reality"
author: "@pweschmidt"
rse: "Stephan Druskat"
phenotype: sdruskat-rse-phenotype.png
excerpt: "Meet Stephan Druskat, RSE at the Alexander von Humboldt University in Berlin, Germany. Listen to Stephan explain his exciting work for research in linguistics and his contributions to making software citable."
date: 2020-07-09 8:30:00
external_media: https://us-rse.org/rse-stories-episodes-1/2020/rse-stories-stephan-druskat-episode-24.mp3
length: 19032316
duration: "00:39:39"
explicit: "no"
resources:
 - name: Stephan Druskat
 - url: https://sdruskat.net
 - name: Stephan Druskat Twitter
   url:  https://twitter.com/stdruskat
 - name: Alexander von Humboldt University
   url: https://www.informatik.hu-berlin.de/en/standardseite-en?set_language=en
 - name: MelaTAMP project (linguistics)
   url: https://www.projekte.hu-berlin.de/en/melatamp
 - name: RSE DE in Digital Humanities
   url: https://dh-rse.github.io
---

> Today we go to Berlin in Germany to meet with Stephan Druskat (@stdruskat). Stephan is a research software engineer at

Reviewer: ```suggestion
Today we go to Berlin in Germany to meet with Stephan Druskat ([@sdruskat](https://github.com/sdruskat)). Stephan is a research software engineer at
```

Reviewer: We have his Twitter in the resources, so an @ (at) in the context of GitHub would  be expected to be his GitHub alias I think!

> the Alexander-von-Humboldt Universität in Berlin (https://www.informatik.hu-berlin.de/).

Reviewer: ```suggestion
the [Alexander-von-Humboldt Universität in Berlin](https://www.informatik.hu-berlin.de/).
```

He has been involved with a lot of exciting research on
linguistics, such as the study of oceanic languages (https://www.projekte.hu-berlin.de/en/melatamp).

> Another focus of his work has been "software citation" and ensuring that software becomes a valid and recognised output of research (https://elib.dlr.de/133021/1/druskat-cise-2019.pdf).

Reviewer: ```suggestion
Another focus of his work has been "software citation" and ensuring that software becomes a valid and recognised output of research [[ref]](https://elib.dlr.de/133021/1/druskat-cise-2019.pdf).
```

> And if that isn't enough, Stephan plays an important part in building the RSE community (https://dh-rse.github.io) in Germany.

Reviewer: ```suggestion
And if that isn't enough, Stephan plays an important part in building the [RSE community in Germany](https://dh-rse.github.io).
```
